### DISCUSSION TITLE
一些建议

### DISCUSSION THREAD

**Discussion by xiaocao666tzh** [Baiyuetribe/kamiFaka#38] [新功能提交]
1、为什么通过联系方式查询订单仅限下单后2小时内有效啊，能改成永久有效吗？
2、希望以后能对接PayPal
3、tg发卡的文档官方文档里面好像没有啊

**Comment by Baiyuetribe** [comment]
最初设计里，联系方式是可以永久查询的，但是很多人反馈圈子很小，容易泄露卡密，因此改成了2小时以内，订单号查询可以永久查看，一般在支付宝或微信的付款记录里可以查看订单号；paypal的支持需要有人参与调试；tg发卡文档稍后补全
